src/components/dialog.js

```diff
@@ -0,0 +1,102 @@
+(() => ({
+  name: 'Dialog',
+  type: 'CONTAINER_COMPONENT',
+  allowedTypes: ['BODY_COMPONENT', 'CONTENT_COMPONENT'],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const { env } = B;
+    const { Dialog } = window.MaterialUI.Core;
+
+    const isEmpty = children.length === 0;
+    const isDev = env === 'dev';
+    const isPristine = isDev && isEmpty;
+    const isContentVisible = isDev && options.isVisibleInDev;
+
+    const classNames = isContentVisible
+      ? [
+          classes.content,
+          isEmpty ? classes.empty : '',
+          isPristine ? classes.pristine : '',
+        ].join(' ')
+      : '';
+
+    const [open, setOpen] = React.useState(false);
+    const devContent = isContentVisible && !isEmpty ? children : 'Dialog';
+
+    useEffect(() => {
+      B.defineFunction('handleDialogOpen', e => {
+        e.preventDefault();
+
```

Comment:
This white line doesn't make the code better readable. Please remove it. Also, for line 35.